postman/Get Broadcast Open Clicks.postman_collection.json

```diff
@@ -0,0 +1,309 @@
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_URL}}/accounts",
+					"host": [
+						"{{base_URL}}"
+					],
+					"path": [
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists on Account",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "4d22c8fa-8695-4098-a275-10658e8aaea5",
+						"exec": [
+							"pm.test(\"Successfully got an listId\", function() {",
+							"    var jsonData = pm.response.json();",
+							"    if (typeof jsonData.total_size == 'number' ) {",
+							"        var listId = Math.floor((Math.random() * jsonData.total_size ))",
+							"        pm.globals.set(\"listId\",jsonData.entries[listId].id) ",
+							"    }",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_URL}}/accounts/{{accountId}}/lists",
+					"host": [
+						"{{base_URL}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Number of Broadcasts on List",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "1bac9b8a-de19-4264-9c19-bdc1ea154a5c",
+						"exec": [
+							"pm.test(\"Successfully number of broadcasts in list\", function() {",
+							"    var broadcast_entry = Math.floor((Math.random() * responseBody ))",
+							"    if (typeof broadcast_entry == 'number' ) {",
+							"        pm.globals.set(\"broadcast_entry\",broadcast_entry )",
+							"    }",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_URL}}/accounts/{{accountId}}/lists/{{listId}}/campaigns?campaign_type=b&ws.op=find&ws.show=total_size",
+					"host": [
+						"{{base_URL}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"campaigns"
+					],
+					"query": [
+						{
+							"key": "campaign_type",
+							"value": "b"
+						},
+						{
+							"key": "ws.op",
+							"value": "find"
+						},
+						{
+							"key": "ws.show",
+							"value": "total_size"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Random Broadcast",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "d2b1b30c-3aec-48cc-9f35-c1ab0683ee45",
+						"exec": [
+							"pm.test(\"Successfully got a random broadcast\", function() {",
+							"    var jsonData = pm.response.json()",
+							"    if (typeof jsonData.entries[pm.globals.get(\"broadcast_entry\")].id == 'number' ) {",
```

Comment:
This could fail if the total number of broadcasts is more than fits on a page (100 by default). I recommend ignoring the total count and picking one of the ones returned on the first page.